stwcs/updatewcs/astrometry_utils.py

```diff
@@ -158,6 +158,10 @@ def updateObs(self, obsname):
         wcsnames = headerlet.get_headerlet_kw_names(obsname, 'wcsname')
 
         headerlets, best_solution_id = self.getObservation(observationID)
+        # Determine whether best_solution_id is simply the pipeline default solution
+        if len(best_solution_id.split('-')) == 1: 
+            best_solution_id = None
```

Comment:
How does this condition actually determine that it is the default pipeline solution?
This implies some naming convention which I am not familiar with.

Reply:
And what is the "default pipeline" solution? Is it the original OPUS WCS?

Reply:
The astrometry database will return an empty string (or None) as the value when no solution has been identified as the 'best' solution.  Also, the WCSNAME for all a priori and a posteriori solutions will start with the 'pipeline default' solutions (WCSNAME=OPUS or IDC_<ipppssoot>) and append the new solution information to them joined by '-'. This is documented in the Drizzlepac readthedocs section on the astrometry solutions (https://drizzlepac.readthedocs.io/en/latest/astrometry.html#interpreting-wcs-names) and in the public pages for this project (https://outerspace.stsci.edu/display/HAP).   

Should this condition be met, it means there is no a priori or archive a posteriori solutions in the database.  In those cases, all solutions from the database will be appended to the exposure being processed and the current distortion model which updatewcs just applied (which I referred to as the 'pipeline default') will remain unchanged as the 'best' solution for subsequent processing.  
I have also accounted for empty return value from the database in commit 4ed7a56.